hadoop-hdds/test-utils/src/main/java/org/apache/ozone/test/GenericTestUtils.java

```diff
@@ -464,4 +469,34 @@ public void write(byte[] buf, int off, int len) {
     }
   }
 
+  public static String getLexicographicallyLowerString(String val) {
+    char[] charVal = val.toCharArray();
+    charVal[charVal.length - 1] -= 1;
+    return String.valueOf(charVal);
+  }
+
+  public static String getLexicographicallyHigherString(String val) {
+    char[] charVal = val.toCharArray();
+    charVal[charVal.length - 1] += 1;
+    return String.valueOf(charVal);
+  }
+
+  public static List<Optional<String>> getTestingBounds(
```

Comment:
nit: Though I see it is used in `hdds-rocks-native` and `rocksdb-checkpoint-differ`, I doubt it will be used anywhere else to qualify to be added in `GenericTestUtils`. I will suggest to add this as util in `hdds-rocks-native` and reuse in `rocksdb-checkpoint-differ`.

Reply:
Sure will do this.

Reply:
done